docs/role-acme.md

```diff
@@ -60,11 +60,17 @@ Feel free to contribute more DNS or HTTP APIs :)
 | acme_staging_directory            | no       | acme-staging-v02.api.letsencrypt.org | Acme directory which will be used for certificate challenge
 | acme_live_directory               | no       | acme-v02.api.letsencrypt.org         | Acme directory which will be used for certificate challenge
 | acme_account_key_path             | no       | $acme_conf_dir                       | Path for account key
+| acme_account_key_size             | no       | 384                                  | Account key size
```

Comment:
if the user chooses "ECC" key type the size is ignored but it needs to be 4096 for a secure RSA key. With the previous setting after changing key type to RSA it would try to create a 384bit RSA key.